<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dsSubmit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>460</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>241</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>241</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>460</width>
       <height>241</height>
      </rect>
     </property>
     <widget class="QLabel" name="shotLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Shot:</string>
      </property>
     </widget>
     <widget class="QLabel" name="RLLabel">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>0</y>
        <width>189</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>RenderLayer:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="rlList">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>20</y>
        <width>220</width>
        <height>221</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="shList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="renderBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>2</y>
        <width>144</width>
        <height>16</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>mayaSoftware</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mayaHardware</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mentalRay</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>vray</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="popupFrame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>5</y>
      <width>210</width>
      <height>241</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="priorityLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prio:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="priotyCombo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="StartFrameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start:</string>
     </property>
    </widget>
    <widget class="QLabel" name="EndFrameText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>74</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>End:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="StartFrameInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>54</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="EndFrameInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>74</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="saveFileCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save File</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="DeleteFramesCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Frames</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="YesButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLabel" name="UserText_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UserText">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="ApproveCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Approve</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="CamText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Cam:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CamPopupCombo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="GICheck">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>USE GI CACHE</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="minSeqDivLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>41</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxSeqDivLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="minSeqDivInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>54</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxSeqDivInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>74</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>452</width>
      <height>32</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>452</width>
       <height>32</height>
      </rect>
     </property>
     <widget class="QPushButton" name="RefreshButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>59</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExplorerButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>59</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Explorer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PublishButton">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>0</y>
        <width>59</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Publish</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ViewerButton">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>0</y>
        <width>59</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>viewer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VrayElemButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>59</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>vRec</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ControlButton">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>0</y>
        <width>59</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>control</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SubmitButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>59</width>
        <height>28</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGI_RL"/>
    <addaction name="actionSGI_RL"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWIKI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionWIKI">
   <property name="text">
    <string>Wiki</string>
   </property>
  </action>
  <action name="actionGI_RL">
   <property name="text">
    <string>CreateGI_RL</string>
   </property>
  </action>
  <action name="actionSGI_RL">
   <property name="text">
    <string>SelectGI_RL</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
